### DISCUSSION TITLE
Release timeline?

### DISCUSSION THREAD

**Discussion by rjmackay** [sensedeep/dynamodb-onetable#124] [General]
I'd love to get recent changes into a release, so I can stop installing direct from github. What timeline do you normally do releases on? (I also noticed you tagged 1.8.1 but didn't actually push a release)

**Comment by mobsense** [comment]
The current release is 1.8.0. 

1.8.1 is pre-release. The process is we aggregate features into a GitHub release flagged as pre-release. But this is just for developers like you who can get directly from GitHub.

We try to push a release as soon as there is a forcing fix. Our goal is to keep the issues list at zero -- so we push pretty often.

I was planning on pushing 1.8.1 today.

**Comment by mobsense** [comment]
Done.

**Reply by rjmackay** [reply]
Awesome. Thank you!
